src/olympia/lib/settings_base.py

```diff
@@ -1533,12 +1533,39 @@ def get_redis_settings(uri):
 
 # This is the signing server for signing files.
 SIGNING_SERVER = ''
+
 # And how long we'll give the server to respond.
 SIGNING_SERVER_TIMEOUT = 10
+
 # Hotfix addons (don't sign those, they're already signed by Mozilla.
 HOTFIX_ADDON_GUIDS = ['firefox-hotfix@mozilla.org',
                       'thunderbird-hotfix@mozilla.org']
 
+AUTOGRAPH_CONFIG = {
+    'server_url': env(
+        'AUTOGRAPH_SERVER_URL',
+        default='http://autograph:8000'),
+    'user_id': env(
+        'AUTOGRAPH_HAWK_USER_ID',
+        default='alice'),
+    'key': env(
+        'AUTOGRAPH_HAWK_KEY',
+        default='fs5wgcer9qj819kfptdlp8gm227ewxnzvsuj9ztycsx08hfhzu'),
```

Comment:
Is this default the actual key?  I assume it's not secret-ish?

Reply:
This is the api key to autograph's dev configuration, which is public https://github.com/mozilla-services/autograph/blob/master/autograph.yaml#L217

Reply:
Jupp, it's only for testing and does not give the signatures any power